hack/tools/release/docker-image-verify.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
```suggestion

set -o errexit
set -o nounset
set -o pipefail

```

Small improvement, but this way it also detects if the REGISTRY variable is set. A simple `make docker-images-verify` will fail without REGISTRY being set and create lots of garbage output :-)

Reply:
Done